src/sv_port.rs

```diff
@@ -332,43 +356,43 @@ fn port_parameter_datatype_ansi(
 
     match datatype {
         Some(RefNode::IntegerVectorType(sv_parser::IntegerVectorType::Logic(_))) => {
-            (Some(SvDataType::Logic), false)
+            return (Some(SvDataType::Logic), false);
```

Comment:
Can this be refactored to use `return` sparingly?
It would be much easier to follow if the idiomatic Rust style is followed (without the `return` keyword).

Reply:
Once the other two changes have been implemented, I will immediately start working on a refactored version that addresses this.

Reply:
Fixed.